script.js

```diff
@@ -0,0 +1,88 @@
+let defaultContainer = document.getElementById("defaultContainer")
+let divColor = document.getElementById("text")
+let dimension = document.getElementById("dimension")
+let container = document.getElementById("container")
+// functions for different kinds of colors
+function red(){
+    divColor.textContent = "RED"
+}
+function yellow(){
+    divColor.textContent = "YELLOW"
+}
+function green(){
+    divColor.textContent = "GREEN"
+}
+function blue(){
+    divColor.textContent = "BLUE"
+}
+function eraser(){
+    divColor.textContent = "ERASER"
+}
+
+for (let j = 1 ; j<=256 ; j++){ 
+    let defaultContent = document.createElement("div"[j])
+    defaultContainer.appendChild(defaultContent)
+    defaultContent.style.width = "25px" 
+    defaultContent.style.height = "25px"
+    defaultContent.addEventListener("mouseover", hover)
+    function hover(){
+        if(divColor.textContent === "BLUE"){
+            defaultContent.style.background = "blue" 
```

Comment:
How about making these colors CSS classes. Then you can just add it or remove it from target elements instead of having to always do `[element].style.background=[color]`

Reply:
I have updated this, kindly check it out if I did the right thing